CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Added rel="noopener noreferrer" in documentation links. [#5197](https://github.com/wazuh/wazuh-kibana-app/pull/5197) [#5274](https://github.com/wazuh/wazuh-kibana-app/pull/5274) [#5298](https://github.com/wazuh/wazuh-kibana-app/pull/5298)
 - Added `ignore` and `restrict` options to Syslog configuration. [#5203](https://github.com/wazuh/wazuh-kibana-app/pull/5203)
+- Added the `extensions.github` and `extensions.office` settings to the default configuration file [#5376](https://github.com/wazuh/wazuh-kibana-app/pull/5376)
```

Comment:
We must add these settings to the documentation: https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/config-file.html

The issues resolved by this pull request will stay blocked until then.

Reply:
I created a pull request to add the settings: https://github.com/wazuh/wazuh-documentation/pull/6036

Reply:
Thanks!